include/torch-mlir/Dialect/Torch/Utils/TorchUpstream.h

```diff
@@ -97,6 +97,16 @@ enum MemoryFormat {
   ChannelsLast3d
 };
 
+//===----------------------------------------------------------------------===//
+// Possible values for `layout` argument in PyTorch ops that support it.
+// Source:
+// https://pytorch.org/docs/stable/tensor_attributes.html#torch.torch.layout
```

Comment:
I think this is not the right source. Looking at pytorch's implementation of empty here:
https://github.com/pytorch/pytorch/blob/3c1c7fd26e94d00cfa50b7e6dab79f7c7988ae40/aten/src/ATen/EmptyTensor.cpp#L98-L106

I think it's this enum that needs to be copied https://github.com/pytorch/pytorch/blob/3c1c7fd26e94d00cfa50b7e6dab79f7c7988ae40/c10/core/Layout.h#L9